scaler/handlers.go

```diff
@@ -60,15 +63,30 @@ func (e *impl) StreamIsActive(
 	}
 }
 
+func getTargetMetric() int64 {
+	targetMetricStr, err := env.Get("KEDA_HTTP_SCALER_TARGET_METRIC")
```

Comment:
@ajanth97 can you read this environment variable in the `main` function [near where all the other configs are read](https://github.com/kedacore/http-add-on/blob/main/scaler/main.go#L22-L40), and then pass the value into this function?

Reply:
I made new changes, please check it